easybuild/easyconfigs/w/WEKA/WEKA-3.6.12-Java-1.7.0_67.eb

```diff
@@ -0,0 +1,37 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+
+easyblock = "Tarball"
+
+name = 'WEKA'
+version = '3.6.12'
+versionsuffix = '-Java-1.7.0_67'
```

Comment:
I suspect Jenkins is complaining about this missing Java version, so yeah, let's bump it?